support/ebpf/bpfdefs.h

```diff
@@ -4,6 +4,9 @@
 #include "bpf_map.h"
 #include "kernel.h"
 
+// with_debug_output is declared in native_stack_trace.ebpf.c
+extern volatile const u32 with_debug_output;
```

Comment:
Why is `volatile` needed here? That generally sounds like having different meaning as `const`. Could `volatile` be just dropped? Maybe it'll also improve ebpf/assembly generation?

Reply:
I have tried things like the following to get it work without volatile:
```
#define BPF_RODATA_VAR()                                     \
    _Pragma("GCC diagnostic push")                                               \
    _Pragma("GCC diagnostic ignored \"-Wignored-attributes\"")                   \
    const u32 with_debug_output __attribute__((section(".rodata"), used)) = 0; \
    _Pragma("GCC diagnostic pop")
```
but the compiler doesn't like it :/